index.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const NR = require('node-resque');
+
+// SET UP THE CONNECTION
+const connectionDetails = {
+    pkg: 'ioredis',
+    host: process.env.REDIS_HOST,
+    password: process.env.REDIS_PASSWORD,
+    port: process.env.REDIS_PORT,
+    database: 0
+};
+
+// DEFINE YOUR WORKER TASKS
+const jobs = {
+    start: {
+        plugins: ['jobLock', 'retry'],
+        pluginOptions: {
+            jobLock: {},
+            retry: {
+                retryLimit: 3,
+                retryDelay: (1000 * 5)
+            }
+        },
+        /* config looks like this
+            executor:
+                name: k8s-vm
+                options:
+                    kubernetes: ...
+            buildConfig:
+                apiUri:
+                buildId:
+                container:
+                token:
+         */
+        perform: (config, callback) => {
+            // eslint-disable-next-line
+            const ExecutorPlugin = require(`screwdriver-executor-${config.executor.name}`);
```

Comment:
> If we want to have dedicated workers across different pods, then we need to set up scheduler to for job allocation. So Dao and I decided just to just have one worker for everything in first pass

Hmmm, I'm confused then. The workers don't do the actual work: they just start a job to run in a different place. You said that "*we need to set up scheduler to for job allocation*": why do you need a scheduler?